cmd/app/main.go

```diff
@@ -46,17 +53,48 @@ func main() {
 
 	sqsClient := sqspkg.NewClient(sqsservice.NewFromConfig(awsCfg), fmt.Sprintf("%s/000000000000/%s", cfg.AWS.EndpointUrl, cfg.AWS.FakeNewsQueueName))
 
+	dbClient, err := gorm.Open(pg.Open(cfg.DB.URL), &gorm.Config{})
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	db, err := postgres.New(dbClient, log)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	err = db.Migrate()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	tokenManager, err := token.NewManager(cfg.SecretKey, "ottct")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	userManager, err := handler.NewAuthManager(db, log, validator.New(), tokenManager)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	entityManager := handler.NewEntityManager(db, log)
+
+	subscriptionsManager, err := handler.NewSubscriptionManager(db, log, validator.New())
```

Comment:
Ovo je duplikat inicijalizacije u [app-u](https://github.com/kordape/ottct-main-service/blob/develop/internal/app/app.go#L28-L56)

Mozes da injectujes iz main-a u [Run](https://github.com/kordape/ottct-main-service/blob/develop/internal/app/app.go#L25) sve ove klijente

Reply:
Ili da pomeris setup za app u main ☺️

Reply:
Slep sam 🤦 vidim da si vec to uradila 😄